pkg/remote/google/repository/cloudresourcemanager.go

```diff
@@ -58,10 +59,10 @@ func (s *cloudResourceManagerRepository) ListProjectsBindings() (map[string]map[
 
 	if len(errorsByProject) > 0 {
 		for project, errval := range errorsByProject {
-			erorsString = erorsString + "Project: " + project + " had the following error: " + errval.Error() + "; "
+			erorString = erorString + fmt.Sprintf("Project: %s had the following error: %s; ", project, errval.Error())
```

Comment:
Not sure to get the end of the template `%s; `, the semicolon is unnecessary here

Reply:
The reason I've put the semicolon is to separate between multiple errors. E.g. 
`For scope projects/123456 on resource TheResource got error: rpc error: code = 123 desc = description; For scope projects/123456 on resource AnotherResource got error: abc`